otherlibs/stdune-unstable/stdune.ml

```diff
@@ -47,6 +47,8 @@ module Digest = Digest
 module Fdecl = Fdecl
 module Unit = Unit
 module Monad = Monad
+module State = State
+module Monad_intf = Monad_intf
```

Comment:
The convention is `module type X = X_int`, there is already `module type Monad = Monad_intf` bellow in the file.

Reply:
Done.

Reply:
It's the other line we should keep, since it follows the current convention.